fdbserver/include/fdbserver/MockGlobalState.h

```diff
@@ -61,12 +61,14 @@ inline bool isStatusTransitionValid(MockShardStatus from, MockShardStatus to) {
 }
 
 class MockStorageServerImpl;
-class MockStorageServer : public IStorageMetricsService {
+class MockStorageServer : public IStorageMetricsService, public ReferenceCounted<MockStorageServer> {
 	friend struct MockGlobalStateTester;
 	friend class MockStorageServerImpl;
 
 	ActorCollection actors;
 
+	CommonStorageCounters counters{ "", "" };
```

Comment:
[Optional code style] When the parameters are not meaningful variables, I think it helps readability to use a pattern like this: `{/*name=*/"", /*id=*/""}`.